server/server.go

```diff
@@ -124,9 +124,6 @@ func New(config Config) (Server, error) {
 	if config.HandlerWrapper == nil {
 		return nil, microerror.Maskf(invalidConfigError, "handler wrapper must not be empty")
 	}
-	if config.ListenAddress == "" {
-		return nil, microerror.Maskf(invalidConfigError, "listen address must not be empty")
-	}
```

Comment:
This check must not be removed. When tests fail they should rather be adapted to have a listen address being configured.